src/code/code_800F9280.c

```diff
@@ -2,9 +2,88 @@
 #include "global.h"
 #include "ultra64/abi.h"
 
-extern u8 sSeqCmdRdPos;
-extern u8 sSeqCmdWrPos;
-extern u8 D_80133410[];
+// bss
+Struct_8016E320 D_8016E320[4][5];
+u8 D_8016E348[4];
+u32 sAudioSeqCmds[0x100];
+unk_D_8016E750 D_8016E750[4];
+u8 D_8016F0E0[0xA0];
+AudioContext gAudioContext;
+void (*D_801755D0)(void);
+s32 D_801755D8[3]; // bss padding
```

Comment:
Building off the previous comments, now it's just these 4 bss variables that I'm not sure exactly where to go, and they could be pushed further down. However, going in order of spec, there's only this file and `audio_init_params.c` left and then that's the end of audio. These 4 can be moved to `audio_init_params.c` and still match. However, both here and in init params, the bss padding that looks like file padding still seems to need to be there to get a match